internal/config/env/configurer.go

```diff
@@ -34,7 +35,8 @@ func New(key string) *Configurer {
 // Configure fetches the values of the env variable for file name and sets that in the config
 func (e *Configurer) Configure(c *config.Config) error {
 	if v, ok := os.LookupEnv(e.key); ok {
-		return yaml.Unmarshal([]byte(v), c)
+		yamlFile, _ := ioutil.ReadFile(v)
```

Comment:
Return error if we can't read it

Reply:
Sure. @kelindar  Have dev tried using YAML as a config file before? The logic before seems not correct.